src/Installer/redist-installer/targets/GenerateLayout.targets

```diff
@@ -460,6 +471,11 @@
           SkipUnchangedFiles="true"
           />
 
+    <Copy SourceFiles="@(BundledLayoutLibraryPackage)"
```

Comment:
Some CI jobs fail here since they don't find the nuget packages (from the `runtime` build) that are supposed to be copied here.

Reply:
I'm not familiar with the different CI jobs.

We could set `/p:BundleNativeAotCompiler=false` on the jobs that aren't doing a full vmr build.

@baronfel @MichaelSimons can you provide some guidance?

Reply:
@baronfel @MichaelSimons @ViktorHofer can we try to get this under way to the VMR?
I would like to avoid surprise issues close to the next preview cut-off.

Reply:
I don't actually know the build enough to quickly see what's the problem - and it looks like the builds have expired at this point. I'll retrigger the failing source-build leg and see if there's something actionable.

Reply:
Might need guidance from the source-build folks here - is it ok to skip the feature in the source-build CI pipeline? is this something that's only relevant for the unified build?

Reply:
The failure in the `dotnet-sdk-public-ci (Build Source-Build (Managed))` leg isn't VMR related. That leg source builds the sdk repository in isolation.

> We could set /p:BundleNativeAotCompiler=false on the jobs that aren't doing a full vmr build.

I don't see any failures in CI related to short stack VMR builds. The sdk-source-build leg builds the full VMR. The error that I see there is: `/vmr/.dotnet/sdk/9.0.100-preview.6.24311.23/NuGet.targets(170,5): error : Object reference not set to an instance of an object. [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj]` when building runtime.

Reply:
@ViktorHofer I'm not sure what log you are looking at.

In https://dev.azure.com/dnceng-public/public/_build/results?buildId=710632&view=logs&j=2f0d093c-1064-5c86-fc5b-b7b1eca8e66a&t=d2b639a6-cb19-5f1f-66fd-8047f66b3026 I see:

```
 /__w/1/s/artifacts/sb/src/src/Installer/redist-installer/targets/GenerateLayout.targets(474,5): error MSB3030: Could not copy the file "/../runtime/Microsoft.NET.ILLink.Tasks.9.0.0-preview.6.24307.2.nupkg" because it was not found. [/__w/1/s/artifacts/sb/src/src/Installer/redist-installer/redist-installer.csproj]
##[error]artifacts/sb/src/src/Installer/redist-installer/targets/GenerateLayout.targets(474,5): error MSB3030: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy the file "/../runtime/Microsoft.NET.ILLink.Tasks.9.0.0-preview.6.24307.2.nupkg" because it was not found.
  /__w/1/s/artifacts/sb/src/src/Installer/redist-installer/targets/GenerateLayout.targets(474,5): error MSB3030: Could not copy the file "/../runtime/Microsoft.DotNet.ILCompiler.9.0.0-preview.6.24307.2.nupkg" because it was not found. [/__w/1/s/artifacts/sb/src/src/Installer/redist-installer/redist-installer.csproj]
##[error]artifacts/sb/src/src/Installer/redist-installer/targets/GenerateLayout.targets(474,5): error MSB3030: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy the file "/../runtime/Microsoft.DotNet.ILCompiler.9.0.0-preview.6.24307.2.nupkg" because it was not found.
```

Reply:
> @ViktorHofer I'm not sure what log you are looking at.

As mentioned, that's not VMR related. This leg only source-builds dotnet/sdk, outside of the VMR.

> I don't see any failures in CI related to short stack VMR builds. The sdk-source-build leg builds the full VMR. The error that I see there is: /vmr/.dotnet/sdk/9.0.100-preview.6.24311.23/NuGet.targets(170,5): error : Object reference not set to an instance of an object. [/vmr/artifacts/source-built-sdks/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj] when building runtime. That seems to be an unrelated error as your changes don't touch runtime or any repo that builds before it.

Here I'm talking about the failure in the `sdk-source-build` leg which builds the **full** VMR.

Reply:
Ack, this one: https://dev.azure.com/dnceng-public/public/_build/results?buildId=710633&view=logs&j=609589e2-4f74-5576-cdb7-914bcaea778b&t=5ba03121-3cea-598c-07fa-53815ace41ff&l=1328

> That seems to be an unrelated error as your changes don't touch runtime or any repo that builds before it.

I also don't see the relation with this PR.

And I have not seen this error when trying these changes in my local vmr builds.

Probably it won't change anything, but I'm going to rebase my PR.